Lib/pydoc.py

```diff
@@ -1997,7 +1997,11 @@ def __repr__(self):
     _GoInteractive = object()
     def __call__(self, request=_GoInteractive):
         if request is not self._GoInteractive:
-            self.help(request)
+            try:
+                self.help(request)
+            except ImportError as e:
+                self.output.write(str(e))
+                self.output.write("\n")
```

Comment:
Would f-strings be better than two `write`s?

```suggestion
                self.output.write(f'{e}\n')
```